pages/common/http-server-upload.md

```diff
@@ -0,0 +1,28 @@
+# http-server-upload
+
+> Zero-configuration command-line HTTP server which provides a lightweight interface to upload files.
+> More information: <https://github.com/crycode-de/http-server-upload>.
+
+- Start a HTTP server on the default port to upload files to the current directory:
```

Comment:
It's `an HTTP`, not `a HTTP`.
Also, don't you mean `upload **from** the default directory`?

Reply:
No, the app uploads file vie an HTML form to the current directory.

Reply:
I still don't get it. How can you upload something to your local machine? If you receive it you download, and if you send it you upload.

Reply:
You can start an HTTP server using this `npm` in your laptop or a cloud server, if you have access to that host&port from another PC you can enter to `192.168.0.10:8080` and then use this form with an `<input type="file" />` to upload files to another PC via a simple HTML form.
I used this mechanism a month ago to recover some files from a Windows 98 machine, I couldn't use a cloud storage because it's not compatible with really obsolete browsers, but this simple HTML form works 10/10.